account_analytic_required/README.rst

```diff
@@ -0,0 +1,63 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+   :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+   :alt: License
+
+=========================
+Account Analytic Required
+=========================
+
+This module adds an option *analytic policy* on account types.
+You have the choice between 3 policies : *always*, *never* and *optional*.
+
+Configuration
+=============
+
+For example, if you want to have an analytic account on all your expenses,
+set the policy to *always* for the account type *expense* ; then, if you
+try to save an account move line with an account of type *expense*
+without analytic account, you will get an error message.
+
+Usage
+=====
+
+.. image:: https://odoo-community.org/website/image/ir.attachment/5784_f2813bd/datas
+   :alt: Try me on Runbot
+   :target: https://runbot.odoo-community.org/runbot/87/9.0
+
+Bug Tracker
+===========
+
+Bugs are tracked on `GitHub Issues
+<https://github.com/OCA/account-analytic/issues>`_. In case of trouble, please
+check there if your issue has already been reported. If you spotted it first,
+help us smashing it by providing a detailed and welcomed feedback.
+
+Credits
+=======
+
+Author
```

Comment:
You should remove this distinction. Everyone is a contributor. Alexis was the first contributor of a long list.